### DISCUSSION TITLE
Server Error - class 'django.db.utils.IntegrityError'

### DISCUSSION THREAD

**Discussion by FabbyoMalta** [netbox-community/netbox#7233] [Q&A] [answered]
Hi, I upgraded from Netbox to version 3.0 but I realized later that this version doesn't support secrets so I downgraded the previous version and then I got this error when trying to create prefixes.
```
<class 'django.db.utils.IntegrityError'>

null value in column "mark_utilized" violates not-null constraint
DETAIL:  Failing row contains (152, 2021-09-09, 2021-09-09 14:57:15.438317+00, 1.1.1.0/30, active, , null, null, null, 9, null, f, {}, 0, 0, null).


Python version: 3.8.10
NetBox version: 2.11.12
```

**Comment by candlerb** [comment, marked as answer]
There's no supported "downgrade" process in Netbox: i.e. a way to revert the database to a 2.11-compatible state.  You need to restore the Postgres database from the database backup you took before upgrading.
